tests/run.rs

```diff
@@ -800,3 +800,122 @@ fn complains_when_expected_an_unmocked_command() -> R<()> {
     )?;
     Ok(())
 }
+
+mod file_mocking {
+    use super::*;
+
+    mod in_ruby {
+        use super::*;
+
+        #[test]
+        fn allows_to_mock_files_existence_in_ruby() -> R<()> {
+            test_run(
+                r##"
+                    |#!/usr/bin/env ruby
+                    |`cat /foo/bar` if File.exists?("/foo/bar")
```

Comment:
I think I would try to simplify this a bit. These integration tests are not meant to provide reasonable little ruby snippets, but just the most simple test case for what they're testing. So maybe
``` ruby
`/bin/true` if File.exists?("/foo/bar")
```
or even better
``` ruby
fail if File.exists?("./foo")
```

Reply:
note that the tests print out `/tmp/test.1Gf6lsxeepoC/file:2:in `<main>': unhandled exception` if i call `fail` as you suggest. any advice on how to hide that? 

i could always fail if the file *doesn't* exist and instead expect `Ok(())`. i think i like that better because then the polarity of the assertion matches (successful if the feature works vs failure if the feature works), but let me know if you prefer the `fail`.

Reply:
No, I don't have a particular preference. `fail` was just an idea. The tests should just be as simple as possible.